text/0049-soc-gpio-peripheral.md

```diff
@@ -0,0 +1,221 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [amaranth-lang/rfcs#0000](https://github.com/amaranth-lang/rfcs/pull/0000)
+- Amaranth Issue: [amaranth-lang/amaranth#0000](https://github.com/amaranth-lang/amaranth/issues/0000)
```

Comment:
If you update the header I'll merge now, otherwise it'll wait until Monday.

Reply:
Done.